packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -197,6 +200,10 @@ export class PieCookieBanner extends LitElement implements CookieBannerProps {
         const descriptionLocaleKey = `preferencesManagement.${id}.description`;
         // Ensure not to display fallback text as description as its expected that some items might not have its own description
         const description = hasDescription && this._localiseText(descriptionLocaleKey);
+        const requiredToggleAllKeys = ['functional', 'personalized', 'analytical'];
```

Comment:
I can imagine us having to look back on this in the future, is it worth a comment to explain why these are here?